[Marlin 2.0 Bugfix Branch new layout] - Compile Error with BLTOUCH and PROBING_HEATERS_OFF

Compiling for MOTHERBOARD BOARD_RAMPS_14_EFB
With BLTOUCH and PROBING_HEATERS_OFF defined compiling fails with:

Arduino: 1.8.4 (Windows 10), Board: "Arduino/Genuino Mega or Mega 2560, ATmega2560 (Mega 2560)"

Stuff deleted...

C:\Users\Dave\AppData\Local\Temp\arduino_build_63964\sketch\src\module\motion.cpp: In function 'void do_homing_move(AxisEnum, float, float)':

C:\Users\Dave\AppData\Local\Temp\arduino_build_63964\sketch\src\module\motion.cpp:888:43: error: 'probing_pause' was not declared in this scope

     if (axis == Z_AXIS) probing_pause(true);

                                           ^

C:\Users\Dave\AppData\Local\Temp\arduino_build_63964\sketch\src\module\motion.cpp:908:44: error: 'probing_pause' was not declared in this scope

     if (axis == Z_AXIS) probing_pause(false);

                                            ^

Using library LiquidCrystal at version 1.0.7 in folder: C:\Program Files (x86)\Arduino\libraries\LiquidCrystal 
exit status 1
Error compiling for board Arduino/Genuino Mega or Mega 2560.

--- a/Marlin/src/module/probe.h
+++ b/Marlin/src/module/probe.h
@@ -47,6 +47,14 @@ float probe_pt(const float &lx, const float &ly, const bool, const uint8_t, cons
   void servo_probe_init();
 #endif
 
+#if QUIET_PROBING
+  void probing_pause(const bool p);
+#endif
+
+#if ENABLED(PROBING_FANS_OFF)
+  void fans_pause(const bool p);
+#endif
+
 #if ENABLED(BLTOUCH)
   void bltouch_command(int angle);
   bool set_bltouch_deployed(const bool deploy);

